package.json

```diff
@@ -17,8 +18,7 @@
   ],
   "browserify": {
     "transform": [
-      "glslify",
-      "bubleify"
```

Comment:
Hmm. You no longer convert es6 -> es5?

Reply:
bubleify does not do async/await https://github.com/Rich-Harris/buble/pull/125, and seems there is no more maintenance.
I wished there was a way for browserify to indicate this dep for main export, but ignore that for running tests.
Probably I should include babelify.